common/src/main/java/tachyon/Constants.java

```diff
@@ -166,6 +166,8 @@
       "tachyon.master.journal.tailer.shutdown.quiet.wait.time.ms";
   public static final String MASTER_JOURNAL_TAILER_SLEEP_TIME_MS =
       "tachyon.master.journal.tailer.sleep.time.ms";
+  public static final String MASTER_JOURNAL_TOOL_STDIN_TIMEOUT_MS =
+      "tachyon.master.journal.tool.stdin.timeout.ms";
```

Comment:
Should this be a configuration value or just a parameter passed into the `readJournal` command?